packages/edit-widgets/src/components/widget-areas/index.js

```diff
@@ -10,8 +10,12 @@ import { withSelect } from '@wordpress/data';
 import WidgetArea from '../widget-area';
 
 function WidgetAreas( { areas } ) {
-	return areas.map( ( area, index ) => (
-		<WidgetArea key={ area.id } area={ area } initialOpen={ index === 0 } />
+	return areas.map( ( { id }, index ) => (
+		<WidgetArea
+			key={ id }
+			widgetAreaId={ id }
```

Comment:
can't this be just `id` as the `widgetArea` is already in the name of the component.